SDK_versioned_docs/version-3.0.0/subgraph-example.md

```diff
@@ -0,0 +1,282 @@
+---
+id: Subgraph examples
+sidebar_position: 3
+---
+
+# Subgraph Query Examples
+You can use the subgraph to make GraphQL queries to fetch data about Uniswap V3. You can fetch data points like the spot price of a token, current liquidity of a pool, volume on a certain day, and much more. Below are some example queries. You can copy and paste them in the [v3 explorer](https://thegraph.com/hosted-service/subgraph/uniswap/uniswap-v3) to get fresh data.
```

Comment:
Can we reformat Sentence 2 and 3 into a bullet list table of contents that links down to each example?

The example queries below illustrate common uses.
* Spot price of a token
* Current liquidity of a pool
* Volume on a certain date
* ...